input/postgres/server_stats.go

```diff
@@ -21,12 +22,76 @@ SELECT
 FROM pg_catalog.pg_control_checkpoint()
 `
 
+const xminHorizonSQL string = `
+SELECT
+COALESCE((
+	SELECT
+		CASE WHEN COALESCE(age(backend_xid), 0) > COALESCE(age(backend_xmin), 0)
+			THEN backend_xid
+			ELSE backend_xmin
+		END
+	FROM pg_stat_activity
```

Comment:
I missed that in my earlier review, but for security reasons, its best practice to explicitly use the `pg_catalog` schema, i.e. `pg_catalog.pg_stat_activity`.

Essentially the threat vector here is that someone could create a `pg_stat_activity` view that calls a function in the public schema (creating objects in the "public" schema is something that a lot more users can do in older Postgres versions, if one doesn't explicitly lock down the schema), and then trick the pganalyze collector user into calling that function, e.g. via this query.

(in practice, because `pg_catalog` is listed first in the search path by default, this is not a practical problem in almost all configurations - the issue only happens when someone explicitly includes `pg_catalog` at the end of their search path)

Reply:
Right, good point. We need to do that for function invocations, too (e.g., `pg_catalog.age`).

Reply:
thanks! addressed, I was also using `age` in the other place so I fixed there too.